docs/usermanual.md

```diff
@@ -313,7 +315,7 @@ in a transformation expression in the IoTAgent). In this cases, when a reverse e
 bidirectional attribute is modified, the IoTAgent sends a command to the original device, with the name defined in the
 reverse expression attribute and the ID of the device (see Commands Syntax, just above).
 
-#### <a name="amqpbinding"/> AMQP binding
+#### AMQP binding
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated